docs/docs/get-started/README.md

```diff
@@ -1,3 +1,106 @@
 # Get Started
 
-VAST is database for security analytics.
+VAST is an embeddable security telemetry engine for structured event data.
+Tailor-made for security operations, VAST is the foundation for many data-driven
+detection and response uses cases, such as operationalizing threat intelligence,
+threat hunting, event contextualization, and advanced detection engineering.
+
+:::tip Wanna give it a try?
+The [quick start guide](/docs/get-started/quick-start-guide) shows you how to
+get up and running and gives a quick tour of the key use cases.
+:::
+
+The documentation has the following key sections:
+
+1. [Setup VAST](/docs/setup-vast) describes how you can download, install, and
+   configure VAST in a variety of environments.
+   👉 *Start here if you want deploy VAST.*
+2. [Use VAST](/docs/use-vast) explains how to work with VAST, e.g., ingesting
+   data, running queries, matching threat intelligence, or integrating it with
+   other security tools.
+   👉 *Go here if you have a running VAST, and want to explore what you can do
+   with it.*
+3. [Understand VAST](/docs/understand-vast) describes the system design goals
+   and architecture, e.g., the actor model as concurrency and distribution
```

Comment:
I think the actor model belongs into the _Develop VAST_ section. While we can mention that VAST uses the actor model and that the major system components comprise of at least one actor each, we should leave it at that level and only describe the major components here. Otherwise we risk losing readers early on in a section that actually is interesting to them.

When first mentioning the actor model, we can point to a page in the _Develop VAST_ section that goes in-depth, but is also appropriately technical.

Reply:
I would disagree here. The actor model is a central part of the system architecture, and this is what this section is about. It's about learning VAST internals, what components exist, how the pieces hold together, etc.

CAF specifics go in *Develop VAST*, but not the high-level actor model design.